### DISCUSSION TITLE
tortoise-orm custom user model

### DISCUSSION THREAD

**Discussion by apan-git** [fastapi-users/fastapi-users#707] [Q&A]
I use the models style given by fastapi_users and use the tortoise ORM migration tool, but I get an error

```python
class User(models.BaseUser):
    name: str
    phone: int


class UserCreate(models.BaseUserCreate):
    pass


class UserUpdate(User, models.BaseUserUpdate):
    pass


class UserModel(TortoiseBaseUserModel):
    name = CharField(max_length=255)
    phone = BigIntField()

    class Meta:
        table = "user"


class UserDB(User, models.BaseUserDB, PydanticModel):
    class Config:
        orm_mode = True
        orig_model = UserModel


user_db = TortoiseUserDatabase(UserDB, UserModel)


from fastapi import FastAPI
from Settings.devSetting import ORM_LINK_CONF
from tortoise.contrib.fastapi import register_tortoise
app = FastAPI()
register_tortoise(
    app,
    # db_url=ORM_LINK_CONF.get_orm_base_conf().get("connections"),
    # modules={"models": ["ORM.models"]},
    config=ORM_LINK_CONF,
    generate_schemas=True,
)

```
Here's the mistake
![image](https://user-images.githubusercontent.com/59251803/131624582-9b6840e0-58ac-4650-896b-04e04f6f455c.png)

I really hope to get help. Thank you very much

**Comment by frankie567** [comment]
Apparently, it looks like a mistake in your Tortoise configuration: you specify that your models live in the module `ORM.models`. It means that it expects to find your Tortoise models in a `models.py` file inside a directory named `ORM`. Is it really the case?

**Reply by apan-git** [reply]
Obviously, I made a mistake. I have corrected it in time. Thank you very much
